operator/roles/kiali-remove/tasks/main.yml

```diff
@@ -26,6 +26,12 @@
   - is_openshift == False
   - is_k8s == False
 
+- name: Set default oauth name when using the openshift strategy
+  set_fact:
+    kiali_vars: "{{ kiali_vars | combine({'auth': {'oauth_name': 'kiali-' + kiali_vars.deployment.namespace}}, recursive=True) }}"
+  when:
+  - kiali_vars.auth.oauth_name == ""
```

Comment:
Based on the name ("...when using the openshift strategy"), I think you want an additional when clause here:

```
- is_openshift == True
```